ARGO/src/RenderSystem.cpp

```diff
@@ -161,6 +170,21 @@ void RenderSystem::renderText(SDL_Renderer* t_renderer, TransformComponent* t_po
 	SDL_SetRenderDrawColor(t_renderer, prevRGBA[0], prevRGBA[1], prevRGBA[2], prevRGBA[3]);
 }
 
+bool RenderSystem::inView(TransformComponent* t_posComp)
+{
+	float left, right, top, down;
+	left = m_focusPoint.x - Utilities::SCREEN_WIDTH * 0.75f;
+	right = m_focusPoint.x + Utilities::SCREEN_WIDTH * 0.6f;
+	top = m_focusPoint.y - Utilities::SCREEN_HEIGHT * 0.75f;
+	down = m_focusPoint.y + Utilities::SCREEN_HEIGHT * 0.6f;
```

Comment:
Why is left .75 of the screens width and right .6
also could hold this value as a variable in the header